server/utils/placementApplications/utils.ts

```diff
@@ -0,0 +1,36 @@
+import { UserDetails } from '../../@types/ui'
+import { createQueryString } from '../utils'
+import { TabItem } from '../tasks/listTable'
+import assessPaths from '../../paths/assess'
+import { hasRole } from '../users'
+
+export const assessmentsTabItems = (user: UserDetails, activeTab?: string): Array<TabItem> => {
```

Comment:
Other than in the tests where is this function used?

Reply:
some issue didnt pushed some files rebased and pushed again